dipy/data/fetcher.py

```diff
@@ -611,33 +635,22 @@ def read_mni_template(contrast="T2"):
     >>> T2_nifti = read_mni_template("T2") # doctest: +SKIP
     Get both files in this order:
     >>> T1_nifti, T2_nifti = read_mni_template(["T1", "T2"]) # doctest: +SKIP
-
     """
-    folder = pjoin(dipy_home, 'mni_template')
+    files, folder = fetch_mni_template()
     file_dict = {"T1":pjoin(folder, 'mni_icbm152_t1_tal_nlin_asym_09a.nii'),
```

Comment:
To be clear, I chose 'a' because that's what @jyeatman was using in his
work on AFQ. Jason - any particular reason to use 'a' rather than 'c'?


On Thu, Aug 6, 2015 at 9:52 AM, Matthew Brett <notifications@github.com>
wrote:

> In dipy/data/fetcher.py
> <https://github.com/nipy/dipy/pull/681#discussion_r36437620>:
>
> >      """
> > -    folder = pjoin(dipy_home, 'mni_template')
> > +    files, folder = fetch_mni_template()
> >      file_dict = {"T1":pjoin(folder, 'mni_icbm152_t1_tal_nlin_asym_09a.nii'),
>
> Ah yes - sorry - you are right about 2009c. Then - do you know the
> difference between 2009a and 2009c? The page mentions only the different
> version numbers for the intensity correction method and different
> 'resampling'.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/681/files#r36437620>.
>